model/lib/extensions/email_address_extension.dart

```diff
@@ -27,6 +27,17 @@ extension EmailAddressExtension on EmailAddress {
     return '';
   }
 
+  String asFullHtmlString() {
```

Comment:
write unit test for this

Reply:
Widget test to make sure HTML content contain enough: `Subject, From, To, Cc, Bcc, Reply to`

Reply:
done